### DISCUSSION TITLE
https://blog.yell0wsuit.page/2024/05/23/whats-new-for-my-website

### DISCUSSION THREAD

**Discussion by giscus** [yell0wsuit/yllst-blog#18] [General]
# https://blog.yell0wsuit.page/2024/05/23/whats-new-for-my-website

https://blog.yell0wsuit.page/2024/05/23/whats-new-for-my-website

**Comment by duanyu5871** [comment]
Hello,
I came across your blog on GitHub and noticed that you have collected many HTML5 games on your website. That must have been quite a laborious process. 

I am also trying to save some games from GamePix to my computer, but I have encountered some strange issues. I can obtain most of the files from the network tab and access them locally via a server set up with Nginx, which has worked successfully. However, when I try to access it from another computer using the external IP, the website can be opened, but it will get stuck on the loading screen and causes the CPU to reach 100% for a long time. I initially thought this might be an anti-cracking measure, and I tried to read through the JS files, but I didn't find anything suspicious in them. The remaining possibility could be the compiled WASM files, but I am not familiar with that content.

I am not sure if I can ask you. I don't expect to easily obtain a solution, but I would like to know if you have ever encountered similar issues or if my speculations are close to the correct answer.

Thank you

**Reply by yell0wsuit** [reply]
That’s weird. I didn’t know about that because most of the time the games would work out of the box (provided that you have sufficient files). Or perhaps the game would randomly hang for some reasons.

I collected around 2 years ago, so things must have changed since then. JS files are easier to debug, but for WebAssembly it’s not unless you can read the instructions. But for JS, sometimes they would either add address detection or obscure the code (I hate the latter the most). I’m pretty much busy with other stuff so I couldn’t think of any solutions, sorry 😢.

**Comment by earlgrayness** [comment]
Hi. I believe that after your newest ui update on the website, the website barely works anymore for some reason??? It shows that there's 0 games on the website and some of the buttons on the website don't really work anymore. i don't know if this is an issue with my system, which i haven't had before, but it it's truly an issue with the website, i believe it should be fixed. I love your website!! thank you :3

**Reply by yell0wsuit** [reply]
Hmm. That’s unfortunate. Have you tried clearing the browser’s cache, or try my site in Incognito/Private browsing mode? Because I tested on my iPhone and it’s working as usual. If it still occurs, can you open the DevTool (use the F12 key) and look for the console to see if any red texts thingy happen and send it back to me?

By the way, I appreciate your continued support! I will try my best to make it up and running like usual.

**Comment by ghost** [comment]
Hi! i would like to ask this, why is Subway Surfers Saint Petersburg map not listed anywhere? it's only shown when searching on Google, and that's how i found this little easter egg lol. If possible, i would like to know the reason of why it's hidden, thanks.

https://yell0wsuit.page/assets/games/subway-surfers-unity/stpetersburg.html
